docs/content/en/docs/clustermgmt/observability/loggingSetup.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Configure Fluent Bit for CloudWatch"
+linkTitle: "Fluent Bit for CloudWatch"
+weight: 4
+#aliases:
+#    /docs/clustermgmt/oservability/
+date: 2023-07-28
+description: >  
+---
+
+Fluent Bit is an open source, multi-platform log processor and forwarder which allows you to collect data/logs from different sources, then unify and send them to multiple destinations. It’s fully compatible with Docker and Kubernetes environments. Due to its lightweight nature, using Fluent Bit as the default log forwarder for EKS Anywhere nodes will allow you to stream application logs into CloudWatch Logs efficiently and reliably.
```

Comment:
"Due to its lightweight nature, using Fluent Bit as the default log forwarder" ... remove "default". 

change "EKS Anywhere nodes" to "EKS Anywhere clusters"